### PR TITLE
feat: add api docs

### LINKED ISSUE #181: Create docs page with api

### PR DESCRIPTION
**Pull request by braianj**

### CODE REVIEW

File: src/components/Layout/UserMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+export default React.memo(function UserMenu() {
+  const l = useFormatMessage()
+
+  return (
+    <Menu
+      onClickSettings={undefined as any}
+      hasActivity={false}
+      onClickActivity={undefined as any}
+      onClickProfile={undefined as any}
```
Comment (2fd) [inline src/components/Layout/UserMenu.tsx:21]: why is this required?

**Review by 2fd — CHANGES_REQUESTED** [review]

File: src/components/Layout/UserMenu.tsx (hunk 1 above)
Comment (braianj) [reply src/components/Layout/UserMenu.tsx:21]: Because `UserMenuProps` has this type, should we change that?

```
export type UserMenuProps = Pick<
  BaseUserMenuProps,
  | 'menuItems'
  | 'hasActivity'
  | 'onClickProfile'
  | 'onClickActivity'
  | 'onClickSettings'
> & {
  i18n?: Partial<UserMenuI18N>
  hideBalance?: boolean
}
```

**Review by 2fd — APPROVED** [review]
